_docs/policies/000-submitting-policy.md

```diff
@@ -0,0 +1,36 @@
+# Purpose
+
+Policy Zero establishes the foundational guidelines for creating, reviewing, and maintaining policies in the `status-go` GitHub repository. This policy aims to create a collaborative, inclusive, and transparent process for defining repository policies, specifically regarding how developers engage with and contribute to the repository.
+
+# Submitting a Policy Proposal
+
+- Any individual MAY propose a new policy.
+- Policy ideas SHOULD be discussed with Core Contributors (CCs) and other community members.
+- All policies MUST be submitted to the `_docs/policies` directory as a pull request (PR) within the `status-go` repository.
+- All policies MUST be in Markdown format
+- Policy file names MUST follow [File name conventions for ADRs](https://github.com/joelparkerhenderson/architecture-decision-record?tab=readme-ov-file#file-name-conventions-for-adrs), e.g. `000-submitting-policy.md`
+
+# Review and Approval Process
+
+The core function of the review and approval process for policy PRs is to reach consensus on any issue and to reflect the range of perspectives within the `status-go` community. Policy submissions must aim to achieve broad community support and give key stakeholders a chance to gain context of the policy requirements.
```

Comment:
I still think that this part if redundant. It basically summarises what's described in the README: consensus, transparency and inclusivity.

But let's see what others say.